lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -26,7 +26,11 @@
 #include <Windows.h>
 #include <shlobj.h>
 #else
-#include <limits.h> /* PATH_MAX */
+#ifdef __APPLE__
+ #include <sys/syslimits.h> // PATH_MAX
+#else
+ #include <limits.h> // PATH_MAX
+#endif
```

Comment:
Could you instead use [llvm::sys::fs::make_absolute](http://llvm.org/docs/doxygen/html/namespacellvm_1_1sys_1_1fs.html#ab6c14b25e5233330251efa8e8ca0c212) to fully get rid of the PATH_MAX macro?